compiler/src/iree/compiler/Codegen/Common/GPU/WorkgroupReordering.cpp

```diff
@@ -229,7 +359,26 @@ struct ReorderWorkgroupsPass final
       llvm::dbgs() << "\n\n";
     });
 
-    if (failed(reorderWorkgroupsInFunc(funcOp, reorderingStrategy, logTile))) {
+    uint32_t numXCDs = 1;
+    if (IREE::HAL::ExecutableTargetAttr targetAttr =
+            IREE::HAL::ExecutableTargetAttr::lookup(funcOp)) {
+      if (DictionaryAttr config = targetAttr.getConfiguration()) {
+        if (IREE::GPU::TargetAttr attr =
+                config.getAs<IREE::GPU::TargetAttr>("iree.gpu.target")) {
```

Comment:
Is there no helper function to look this up? Maybe this https://github.com/iree-org/iree/blob/d174e8bcec9f221082511c67111b3f995bdd54a0/compiler/src/iree/compiler/Codegen/Utils/GPUUtils.h#L153 or something in https://github.com/iree-org/iree/blob/main/compiler/src/iree/compiler/Codegen/Utils/Utils.h

Reply:
sure, I will have see whether it is doable